api/src/main/java/io/opentelemetry/metrics/InstrumentWithBind.java

```diff
@@ -20,9 +20,9 @@
  * Base interface for all metrics with bounds defined in this package.
  *
  * @param <B> the specific type of Bound Instrument this instrument can provide.
- * @since 0.1.0
+ * @since 0.3.0
  */
-public interface InstrumentWithBinding<B> extends Instrument {
+public interface InstrumentWithBind<B extends InstrumentWithBind.Bound> extends Instrument {
```

Comment:
I think we should stick with the existing name.

Reply:
Done.